### PR TITLE
Component/icon

### PR DESCRIPTION
**Pull request by HQCuong**

### CODE REVIEW

File: src/components/Icon/Icon.vue [hunk 1]
```diff
@@ -0,0 +1,80 @@
+<template lang="pug">
+span(:class="className")
+  Text(
+    v-if="accessibilityLabel",
+    as="span",
+  )
```
Comment (haild173094) [inline src/components/Icon/Icon.vue:6]: Is this missing content of Text here :thinking: ?

File: docs/components/Icon/Colored.vue [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<template>
+  <Icon :source=CirclePlusMinor tone="base"/>
```
Comment (juzser) [inline docs/components/Icon/Colored.vue:2]: missing double quote `"` in source?

File: src/components/Icon/Icon.vue (hunk 1 above)
Comment (juzser) [reply src/components/Icon/Icon.vue:6]: Also missing `variant` as required property

File: src/components/Icon/Icon.vue [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+  div(
+    v-if="sourceType === 'placeholder'",
+    :class="styles.placeholder",
+  )
+
+  img(
+    v-if="sourceType === 'external'",
+    alt="",
```
Comment (juzser) [inline src/components/Icon/Icon.vue:23]: ?

File: tsconfig.json [hunk 4]
```diff
@@ -30,17 +30,15 @@
     "allowSyntheticDefaultImports": true
   },
   "include": [
+    "docs/components/Icon/*.vue",
```
Comment (juzser) [inline tsconfig.json:33]: Why include only Icon? and remove exclude?

**Review by juzser — CHANGES_REQUESTED** [review]

File: src/components/Icon/Icon.vue (hunk 3 above)
Comment (HQCuong) [reply src/components/Icon/Icon.vue:23]: In React version alt attr is not provided

Comment (HQCuong) [reply src/components/Icon/Icon.vue:23]: Screenshot from 2023-12-28 09-55-54

File: src/components/Icon/Icon.vue (hunk 1 above)
Comment (HQCuong) [reply src/components/Icon/Icon.vue]: Text component - New Version: variant props is not required

**Review by haild173094 — APPROVED** [review]

**Review by juzser — APPROVED** [review]

**Pull request merged**